common/constants.ts

```diff
@@ -1,2 +1,8 @@
 export const DEMO_SITE_SIZE_LIMIT_GB = 2;
 export const DEMO_SITE_SIZE_LIMIT_BYTES = DEMO_SITE_SIZE_LIMIT_GB * 1024 * 1024 * 1024; // 2GB
+export const HOUR_MS = 1000 * 60 * 60;
+export const DAY_MS = HOUR_MS * 24;
+// AI Assistant constants
+// IMPORTANT: When updating this value, we need to update the string located in `AIClearHistoryReminder` component.
+// Reference: https://github.com/Automattic/studio/blob/3dd5c58cdb7998e458d191e508e8e859177225a9/src/components/ai-clear-history-reminder.tsx#L78
+export const CLEAR_HISTORY_REMINDER_TIME = 2 * HOUR_MS; // In milliseconds
```

Comment:
This doesn't strike me as a constant that needs to live in the `common` directory 🤔 It's only used in `src/components`

Reply:
Having an alias import (like `common/constant`) in `src/constants` was causing some issues with the `postinstall` scripts.
I've made the adjustments in https://github.com/Automattic/studio/pull/1222/commits/c554a7dac118ad3dda573600d36139172400ce11 that seems to be an "ok workaround"
Let me know which one you prefer.

Reply:
Ugh, that's frustrating. Do you know which scripts was causing trouble? The latest changes LGTM 👍

Reply:
I think it was this one: https://github.com/Automattic/studio/blob/trunk/scripts/download-wp-server-files.ts#L5
I don't think the constant that we are importing in that script is used anywhere else, so we could also change the script, but it will always be a potential issue, so I think it's best to just reference it and have a relative import.